tests/sentry/api/endpoints/test_project_repo_path_parsing.py

```diff
@@ -197,8 +197,8 @@ def test_basic(self):
             "integrationId": self.integration.id,
             "repositoryId": self.repo.id,
             "provider": "github",
-            "stackRoot": "",
-            "sourceRoot": "src/",
+            "stackRoot": "sentry/",
+            "sourceRoot": "src/sentry/",
```

Comment:
This is confusing, I know. Take a look at this modal to try and understand how `stackRoot` and `sourceRoot` are used for a code mapping, and why the expected behavior is what it is: 

![image](https://github.com/getsentry/sentry/assets/55160142/1ade1459-ceab-4888-81ac-f644ac6de15e)
![image](https://github.com/getsentry/sentry/assets/55160142/aa43c212-6030-475c-a118-8151feb1eed5)

What this is saying is: suppose our **stack trace** is: `sentry/api/endpoints/project_stacktrace_link.py`, and we know that the github **source path** to this file is: `src/sentry/api/endpoints/project_stacktrace_link.py`. The correct `stackRoot` and `sourceRoot` is defined by being able to replace the `stackRoot` in the stack trace with the `sourceRoot` to yield the source path.

Reply:
This makes sense 👍🏻